src/tools/align/state/alignFormReducer.ts

```diff
@@ -70,30 +88,14 @@ export const alignFormParsedSequencesReducer = (
   };
 
   // Set Job Name, if user didn't already set
-  let potentialJobName = '';
-  if (!formValues[AlignFields.name].userSelected) {
-    // if the user didn't manually change the title, autofill it
-    const firstName = parsedSequences.find((item) => item.name)?.name;
-    if (firstName) {
-      potentialJobName = firstName;
-      if (parsedSequences.length > 1) {
-        potentialJobName += ` +${parsedSequences.length - 1}`;
-      }
-    } else if (parsedSequences.length) {
-      potentialJobName = `${parsedSequences.length} ${pluralise(
-        'sequence',
-        parsedSequences.length
-      )}`;
-    }
-  }
-
   const name =
     formValues[AlignFields.name].userSelected &&
     formValues[AlignFields.name].selected
       ? formValues[AlignFields.name]
       : {
           ...formValues[AlignFields.name],
-          selected: potentialJobName,
+          userSelected: false,
```

Comment:
This is necessary in the following scenario:
1. User selected a name
2. Then reset it
3. then provide sequence info
4. Name is not populated as userSelected is 'true' at that point and name is empty.
Right?